src/library/libraryview.h

```diff
@@ -26,6 +26,8 @@ class LibraryView {
     virtual void slotAddToAutoDJBottom() {};
     virtual void slotAddToAutoDJTop() {};
     virtual void slotAddToAutoDJReplace() {};
+    virtual void saveCurrentViewState(){};
```

Comment:
Formatting? This is not an initialization and functions blocks should not be terminated by semicolon.

Reply:
I see that it should be `virtual void saveCurrentViewState() {};`, and I somewhat understand that this is no initialization -- yet I do _not_ understand why I get linker errors as soon as the `{}` are removed. :man_shrugging:  advanced c++ skills still missing...

Reply:
clang-format refuses to add the whitespace though

Reply:
The empty implementation `{}` is still needed if you don't declare them as *pure virtual* by ` = 0;`. Only the semicolon is redundant.

Reply:
Thanks, got it.
fixing it for the new functions only would look weird, so I'd fix it for all here. ?

I guess you'll find other issues during your full review, so I'm not in hurry with this one here..

Reply:
clang enforces a linebreak in between `{}`...